src/babel-plugin-transform-sx-tailwind/src/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -147,6 +147,23 @@ const __styles_3xR9dL = sx.create({
 });
 
 
+`;
+
+exports[`transform SX Tailwind invalid-utility-class.js: invalid-utility-class.js 1`] = `
+
+// @flow
+
+import type { Node } from 'react';
+import { tailwind } from '@adeira/sx-tailwind';
+
+export default function Example(): Node {
+  return <div className={tailwind('bg-bleu-500')}>Button</div>;
+}
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+Error: /Users/michalsanger/projects/adeira/universe/src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/error/invalid-utility-class.js: Unknow utility name: bg-bleu-500
```

Comment:
This should be normalized to some path which will work for everyone.

Reply:
https://github.com/babel-utils/babel-plugin-tester/pull/80